src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/NamedEntityRecognitionSkill.cs

```diff
@@ -0,0 +1,95 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Text analytics named entity recognition.
+    /// <see
+    /// href="https://docs.microsoft.com/azure/search/cognitive-search-skill-named-entity-recognition"
+    /// />
+    /// </summary>
+    [Newtonsoft.Json.JsonObject("#Microsoft.Skills.Text.NamedEntityRecognitionSkill")]
+    public partial class NamedEntityRecognitionSkill : Skill
+    {
+        /// <summary>
+        /// Initializes a new instance of the NamedEntityRecognitionSkill
+        /// class.
+        /// </summary>
+        public NamedEntityRecognitionSkill()
+        {
+            CustomInit();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the NamedEntityRecognitionSkill
+        /// class.
+        /// </summary>
+        /// <param name="description">The description of the skill which
+        /// describes the inputs, outputs, and usage of the skill.</param>
+        /// <param name="context">The context of the skill.</param>
+        /// <param name="inputs">The inputs of the skill.</param>
+        /// <param name="outputs">The outputs of the skill.</param>
+        /// <param name="categories">A list of named entity categories.</param>
```

Comment:
What does "fixed" mean in this context?

Reply:
Static, predefined.  NER supports the 3 built-in categories.  If you want something else, like felon mug shots, you would have to do a custom Web API skill.

Reply:
Makes sense, thanks. It confused me because the fact that categories are fixed seemed obvious from the type of the parameter, so I thought you meant a fixed-length list. I can imagine other developers being similarly confused by “fixed” in this context.

Reply:
I think I agree with Bruce on this one.  "fixed" would confuse me in this case.

Reply:
Based on the discussion here, I will put a "Won't fix" here for now. :) Thanks.